src/client/table_client.rs

```diff
@@ -1520,6 +1492,12 @@ fn build_runtime(name: &str, threads_num: usize) -> runtime::Runtime {
 
 fn build_obkv_runtimes(config: &ClientConfig) -> ObClientRuntimes {
     ObClientRuntimes {
+        batch_op_runtime: Arc::new(build_runtime(
+            "ob-batch-executor",
+            config.batch_op_thread_num,
+        )),
```

Comment:
I guess this runtime is not necessary in the async context, how about remove it?

Reply:
Fixed, we use bg_runtime to deal with a batch operation. It is necessary to create a runtime to process those requests, since batch operation may operate data from different partitions and servers. Hence, we need to seed packages simultaneously and aggregate the results.